lib/ssz_ex.ex

```diff
@@ -205,9 +203,10 @@ defmodule LambdaEthereumConsensus.SszEx do
   @spec hash_tree_root(non_neg_integer, {:int, non_neg_integer}) :: {:ok, Types.root()}
   def hash_tree_root(value, {:int, size}), do: {:ok, pack(value, {:int, size})}
 
-  @spec hash_tree_root(binary, {:bytes, non_neg_integer}) :: {:ok, Types.root()}
-  def hash_tree_root(value, {:bytes, size}) do
-    packed_chunks = pack(value, {:bytes, size})
+  @spec hash_tree_root(binary, {:byte_list | :byte_vector, non_neg_integer}) ::
+          {:ok, Types.root()}
+  def hash_tree_root(value, {type, _size}) when type in [:byte_list, :byte_vector] do
```

Comment:
The original code uses `:bytes` only. Wouldn't this need to include a length mix-in for the `:byte_list` case?